packages/firebase_vertexai/firebase_vertexai/lib/src/api.dart

```diff
@@ -591,7 +591,12 @@ enum HarmBlockThreshold {
   high('BLOCK_ONLY_HIGH'),
 
   /// Always show regardless of probability of unsafe content.
-  none('BLOCK_NONE');
+  none('BLOCK_NONE'),
+
+  /// All content is allowed regardless of harm
```

Comment:
nit: End in '.', and capitalize Metadata?

Reply:
done